src/languages/lsl.js

```diff
@@ -0,0 +1,90 @@
+            },
+            {
+                begin: '\\b(?:FALSE|TRUE)\\b'
```

Comment:
These feel like 'literal' too. And probably some others too, I can't really say as I don't know the language :-) But the rule of thumb is simple: built-in constant values usually go as 'literal', and every other kind of built-in is 'built_in'.